rails_programming/rails_basics/deployment.md

```diff
@@ -143,3 +153,14 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Heroku Custom Domains](https://devcenter.heroku.com/articles/custom-domains) help file
 * [Converting from PostgreSQL to SQLite3](http://manuel.manuelles.nl/blog/2012/01/18/convert-postgresql-to-sqlite/)
 * [Webinar on Using Heroku in Production](https://blog.heroku.com/archives/2013/7/11/running-production-apps-on-heroku)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="#dyno-knowledge-check">What are dynos? how do they work?</a>
+* <a class="knowledge-check-link" href="#management-knowledge-check">What are the two ways you can manage your application with heroku?</a>
```

Comment:
Could we remove this one also? The part where it's mentioned doesn't really dive into it enough to make it worth a knowledge check. It's more just one of those good-to-know things than a critical thing as learners already know about Heroku on the command line.